docker-compose-dev.yml

```diff
@@ -1,6 +1,12 @@
 version: '3'
 
 services:
+  cron:
+    build:
+      context: .
+      dockerfile: Dockerfile_cron
+    environment:
+      - CRON_MINUTE=30 * * * * root curl http://web:5000/check_reservation_status_db
```

Comment:
Why you are using that curl here? We should only call flask command here.

Reply:
I use REST api because it's work better than flask command, really.